src/xmldocvisitor.cpp

```diff
@@ -323,19 +403,79 @@ void XmlDocVisitor::visit(DocVerbatim *s)
       m_t << s->text();
       break;
     case DocVerbatim::Dot:
-      visitPreStart(m_t, "dot", s->hasCaption(), this, s->children(), QCString(""), FALSE, DocImage::Html, s->width(), s->height());
-      filter(s->text());
-      visitPostEnd(m_t, "dot");
+      {
+        static int dotindex = 1;
```

Comment:
Static counters will give unpredictable output if used in a multi-threaded context (i.e. `NUM_PROC_THREADS>1`).

Reply:
I took this code straight away from the other `*docvisitor.cpp` code, (see e.g `htmldocvisitor.cpp` and search for `static int dotindex`.  It is a for the output format specific "global" counter.

@doxygen What do you propose?